### ISSUE TITLE
Word wrapping in blockquotes

### ISSUE THREAD

**Issue by anqxyr** [SublimeText-Markdown/MarkdownEditing#203]
Currently, when a line in a blockquote is wrapped, it visually breaks the left border, like [that](http://i.imgur.com/qGDorIa.png?1). It would be nice if it displayed as a single continuous blockquote instead.

**Comment by felixhao28** [comment]
Post the image in case the url expires.
![image](https://cloud.githubusercontent.com/assets/2417391/6570651/190f8fe8-c739-11e4-9def-07e84b9d50d6.png)

**Comment by felixhao28** [comment]
I don't think it is possible to fix this problem unless we adopt another style of highlight. We can only change the display style of existing characters (in this case, the background color of ">" symbol). Anyone with a better idea is encouraged to discuss here.

**Comment by maliayas** [comment]
This can only be done by ST. We can keep the issue open.
